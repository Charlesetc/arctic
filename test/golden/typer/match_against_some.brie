define main [
  let x = Some 2
  match x [
    Some a -> a
    None -> 0
  ]
]
