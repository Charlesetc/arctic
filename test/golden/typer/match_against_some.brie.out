root[(ident(define):: ident(main):: block[][(ident(define):: ident(x):: (ident(Some):: ident(2)::Integer)::[^ Some Integer ])::, (ident(match):: ident(x)::[^ Some Integer ] block[][(ident(->):: (ident(Some):: ident(a)::Integer):: (ident(a)::Integer)::Integer)::, (ident(->):: ident(None):: (ident(0)::)::)::]::)::Integer]::Function)::]::